tests/terraform_main_test.go

```diff
@@ -0,0 +1,33 @@
+package tests
+
+import (
+	"os"
+	"testing"
+
+	"github.com/gruntwork-io/terratest/modules/terraform"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestTerraformEksCluster(t *testing.T) {
+	// Construct the terraform options with default retryable errors to handle the most common
+	// retryable errors in terraform testing.
+	terraformOptions := terraform.WithDefaultRetryableErrors(t, &terraform.Options{
+		// Set the path to the Terraform code that will be tested.
+		TerraformDir: "../",
+	})
+
+	// Clean up resources with "terraform destroy" at the end of the test.
+	if os.Getenv("DISABLE_DESTROY") != "" {
```

Comment:
Reverse logic so that we destroy unless `DISABLE_DESTROY` is non-empty
```suggestion
	if os.Getenv("DISABLE_DESTROY") == "" {
```